src/ai/backend/manager/models/vfolder.py

```diff
@@ -82,8 +109,22 @@ class VFolderInvitationState(str, enum.Enum):
     sa.Column('creator', sa.String(length=128), nullable=True),
     # For unmanaged vFolder only.
     sa.Column('unmanaged_path', sa.String(length=512), nullable=True),
+    sa.Column('ownership_type', EnumValueType(VFolderOwnershipType),
+              default=VFolderOwnershipType.USER, nullable=False),
     sa.Column('user', GUID, sa.ForeignKey('users.uuid'), nullable=True),
     sa.Column('group', GUID, sa.ForeignKey('groups.id'), nullable=True),
+
+    # Note: Custom constraints should be manually defined during alembic migration.
+    #       This is just for reference.
```

Comment:
Explicit SQLAlchemy declaration is required for "oneshot" schema creation, not only for reference. :wink:

Reply:
Custom constraints will be set by migrating from existing DB, but I'm having trouble to set them directly in the table definition. I'll keep trying to resolve this. Any ideas are welcome.

Reply:
I checked that those constraints are placed successfully by executing `install-dev.sh` script. I think this problem is resolved.